python/tests/image/test_imageIO.py

```diff
@@ -72,71 +74,59 @@ def tearDownClass(cls):
         super(TestReadImages, cls).tearDownClass()
         cls.binaryFilesMock = None
 
-    def test_decodeImage(self):
-        badImg = imageIO._decodeImage(b"xxx")
-        self.assertIsNone(badImg)
-        imgRow = imageIO._decodeImage(pngData)
-        self.assertIsNotNone(imgRow)
-        self.assertEqual(len(imgRow), len(imageIO.imageSchema.names))
-        for n in imageIO.imageSchema.names:
-            imgRow[n]
-
     def test_resize(self):
         imgAsRow = imageIO.imageArrayToStruct(array)
-        smaller = imageIO._resizeFunction([4, 5])
+        imgAsPIL = PIL.Image.fromarray(obj=imageIO._reverseChannels(array)).resize((5,4))
```

Comment:
Clearer reordering of lines (for readability):
```
        self.assertRaises(ValueError, imageIO.createResizeImageUDF, [1, 2, 3])

        make_smaller = imageIO.createResizeImageUDF([4, 5]).func
        imgAsRow = imageIO.imageArrayToStruct(array)
        smallerImg = make_smaller(imgAsRow)
        self.assertEqual(smallerImg.height, 4)
        self.assertEqual(smallerImg.width, 5)

        # Compare to PIL resizing
        imgAsPIL = PIL.Image.fromarray(obj=imageIO._reverseChannels(array)).resize((5,4))
        smallerAry = imageIO._reverseChannels(np.asarray(imgAsPIL))
        np.testing.assert_array_equal(smallerAry, imageIO.imageStructToArray(smallerImg))

        # I'm not sure what the following is testing so I don't know where they should go yet
        for n in ImageSchema.imageSchema['image'].dataType.names:
            smallerImg[n]
        sameImage = imageIO.createResizeImageUDF((imgAsRow.height, imgAsRow.width)).func(imgAsRow)
        self.assertEqual(imgAsRow, sameImage)
```

Reply:
Sure I'll change the order. I believe the last part of the test tests that the image has the correct schema which this might not be the best way to test. I left it in since it was there before.